Automattic/wp-calypso
# Updating image behaviors for sell flow onboarding

#### Changes proposed in this Pull Request

* This PR updates several steps in the "Sell" flavor of the onboarding flow.
* It removes the image from the `store-features` step ("'Set up your store") and adds a new image to replace the existing one for the `site-options` step for this intent (store name and tagline).

![image](https://user-images.githubusercontent.com/13437011/153951923-55e337c4-459f-483f-97f6-8f289fcc2b7b.png)


![image](https://user-images.githubusercontent.com/13437011/153951887-99dc59af-bc00-4337-acdf-ef3b8b2c4fa5.png)


#### Testing instructions



* Check out this PR to local calypso and navigate to `/start`.
* Navigate through the onboarding flow, selecting "Sell" on the intent screen.
* Verify that the site options step has the new image and the store features step has no image.



Fixes #61026
